src/components/ErrorPanel/index.jsx

```diff
@@ -0,0 +1,82 @@
+import { Component } from 'react';
+import { instanceOf, oneOfType, string } from 'prop-types';
+import classNames from 'classnames';
+import Markdown from 'react-markdown';
+import { withStyles } from '@material-ui/core/styles';
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import Paper from '@material-ui/core/Paper';
+import ChevronDownIcon from 'mdi-react/ChevronDownIcon';
+import ErrorBox from './ErrorBox';
+
+@withStyles(theme => ({
+  paper: {
+    padding: `0 ${theme.spacing.double}px`,
```

Comment:
These theme values are custom, i.e. `theme.spacing.double` is a custom addition we put in taskcluster-web. You'll want to copy these.

https://github.com/taskcluster/taskcluster-web/blob/master/src/theme.js#L71-L76